src/libraries/System.Formats.Asn1/tests/Writer/WriteObjectIdentifier.cs

```diff
@@ -229,6 +229,8 @@ public void VerifyWriteObjectIdentifier_ConstructedIgnored(AsnEncodingRules rule
                 new object[] { "Invalid character second position", AsnEncodingRules.CER, "0,1.23" },
                 new object[] { "Invalid character second rid", AsnEncodingRules.DER, "0.1q.23" },
                 new object[] { "Invalid character third rid", AsnEncodingRules.BER, "0.1.23q" },
+                new object[] { "Invalid second RID for first arc 0", AsnEncodingRules.DER, "0.40.1.2.3" },
```

Comment:
Positive test already exists for arc 2.

https://github.com/dotnet/runtime/blob/8575df9d294a25db06fdd138ca0ae5361a8ce5e2/src/libraries/System.Formats.Asn1/tests/Writer/WriteObjectIdentifier.cs#L189